stubs/Model.stub

```diff
@@ -7,6 +7,21 @@ namespace Illuminate\Database\Eloquent;
  */
 abstract class Model implements \JsonSerializable, \ArrayAccess
 {
+    /**
+     * @var array<string, string|class-string>
```

Comment:
`string|class-string` is just `string` Can you change this to `string`?

Reply:
```suggestion
     * @var array<string, string>
```

Reply:
Sure. Will handle it at the evening